### PR TITLE
Added ruby, ruby on rails and rust hindi tutorials in free-courses-hindi.md (request review from urdu speaker)

### PR DESCRIPTION
**Pull request by Darpana-iitgn**
## What does this PR do?
Add resources

## For resources
All 3 resources are youtube playlists which start from basics and go upto advanced topics of Ruby, RoR and Rust.

### Why is this valuable (or not)?
The videos will help beginners develop a good foundation.

### How do we know it's really free?
These are youtube playlists which can be accessed freely by anyone.

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: courses/free-courses-hi.md [hunk 1]
```diff
@@ -742,16 +742,20 @@
...
 * [Complete Ruby Programming Course \|\| Hindi](https://www.youtube.com/playlist?list=PLgPJX9sVy92y_ENwi_4hMCpDtQFnazFPp) - CS Geeks
 * [Learn Ruby in 45 Minutes \| Hindi](https://www.youtube.com/watch?v=3V9a_WYEQPA) - CS Geeks
+* [Ruby Tutorial for Beginners (Hindi)](https://www.youtube.com/playlist?list=PLgPJX9sVy92yefe1xmyxgcyXjxmLHsSEV) - CS Geeks
 
 
 #### Ruby on Rails
 
 * [Full Stack Development in Ruby on Rails - Hindi](https://www.youtube.com/playlist?list=PLSfx1NJkuWPWlVjFy5datW4Y-54ltIFw7) - APPSIMPACT Academy Hindi
 * [Ruby on Rails 5 Tutorial(Hindi)](https://www.youtube.com/playlist?list=PLgPJX9sVy92yV7Qt6_8ElC9paGWdtdIbb) - CS Geeks
+* [Ruby on Rails Course \| Urdu / Hindi](https://www.youtube.com/playlist?list=PL9WbyKqkuCAYT-IPvo5PxR_hijMuR1RCB) - Code with Naqvi
```
Comment (eshellman) [inline courses/free-courses-hi.md:752]: we have an Urdu list - this probably belongs there

**Review by eshellman — CHANGES_REQUESTED** [review]
movr 1 course to urdu list or explain why not

File: courses/free-courses-hi.md (hunk 1 above)
Comment (Darpana-iitgn) [reply courses/free-courses-hi.md:752]: I understand Hindi and after seeing the videos in this playlist, its obvious to me that native Hindi speakers will be able to understand all the videos in the playlist. I don't speak Urdu so i am not sure if people who only speak Urdu would be able to understand, which is why I only kept it in the Hindi playlist. The main language used in the videos is surely Hindi.

Comment (HRIDYANSHU054) [reply courses/free-courses-hi.md:752]: I think it should be listed in both Urdu and Hindi resource files

Comment (eshellman) [reply courses/free-courses-hi.md:752]: I guess we need review from an urdu speaker.

Comment (Darpana-iitgn) [reply courses/free-courses-hi.md:752]: So should I add it in the Urdu file too?

Comment (eshellman) [reply courses/free-courses-hi.md:752]: no we do not duplicate items across lists

**Comment by Ajay-singh1** [comment]
@eshellman @Darpana-iitgn The language used in the playlist is Hindi here and I think it should belong in the Hindi File.I have checked the Urdu file too and found out many courses which have the title as Hindi / Urdu uses Hindi as a language in their youtube videos.

**Comment by Darpana-iitgn** [comment]
@Ajay-singh1 Thank you for your response. I will keep the playlist in the Hindi file.

**Comment by eshellman** [comment]
My understanding is that the big difference between hindu and urdu is the script they're written in, but perhaps all the written bits here are english? In that case we should just keep one copy in the hindu list. But if there's any urdu script, it should go in urdu, right? Thank you both for looking at this!

**Comment by Darpana-iitgn** [comment]
There is only english script no Urdu script

**Comment by Darpana-iitgn** [comment]
@eshellman now there are no further changes needed in the PR so you can merge it.

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**